internal/nearcache/codec.go

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
Isn't this autogenerated as well? Why is this not in proto/codec?

Reply:
No, it's not autogenerated on the Java client as well.